lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -112,7 +111,6 @@ impl<TInvertedIndex: InvertedIndex> SparseVectorIndex<TInvertedIndex> {
         // the underlying upsert operation does not guarantee that the indexed vector count is correct
         // so we set the indexed vector count to the number of points we have seen
         ram_index.vector_count = index_point_count;
-        // TODO(sparse) this operation loads the entire index into memory which can cause OOM on large storage
```

Comment:
It's save to remove this TODO. because while index build we can store whole index in RAM (HNSW does the same)